copier/main.py

```diff
@@ -290,6 +301,9 @@ def update_diff(conf: ConfigData) -> None:
             )
         }
     )
+    # At this point we'll run tasks in the current project:
+    # copy_local() needs to pass "update" as operation
+    os.environ["OPERATION"] = "update"
```

Comment:
This is too much, you're altering the environment variables for all the process.

You better do:

```python
with local.env(OPERATION='update'):
   ...
```

Docs: https://plumbum.readthedocs.io/en/latest/local_machine.html#environment

Reply:
We could also consider passing an `operation` argument to `copy_local`, defaulting to `copy`, as it's the only place we use `os.environ.get("OPERATION")`.